sdk/storage/AzureStorageBlob/Source/Operation/ResumableOperationQueue.swift

```diff
@@ -105,44 +99,21 @@ internal class ResumableOperationQueue {
                         }
                         operation.internalState = .complete
                     }
-                    self.operations.append(operation)
                 }
                 operationQueue.addOperations(filteredOps, waitUntilFinished: false)
             }
         }
     }
 
-    public func cancel(_ operation: ResumableOperation) {
+    func cancel(_ operation: ResumableOperation) {
         operation.cancel()
     }
 
-    public func clear() {
+    func clear() {
         operationQueue.cancelAllOperations()
-        operations.removeAll()
     }
 
-    internal func removeFromDataStore(_ operation: ResumableOperation) {
-        guard let index = operations.firstIndex(of: operation) else { return }
-        operations.remove(at: index)
-    }
-
-    public func pause(_ operation: ResumableOperation) {
+    func pause(_ operation: ResumableOperation) {
         operation.pause()
-        removeFromDataStore(operation)
-    }
-
-    public func remove(_ operation: ResumableOperation) {
```

Comment:
With the removal of the "data source" aspect, all this did was call cancel, so I just removed it and updated references of `remove(...)` to `cancel(...)`

Reply:
One thing I would like to clean up eventually for the public API is that transfers also have a `remove()` method which I would have a hard time explaining to a customer when they should be calling it. It feels like an implementation detail, maybe we can get rid of that for transfers in a future PR.